### PR TITLE
2 - Friends_List Repository

### PR DESCRIPTION
**Pull request by TriAnh1995**
Initializing friends_list Repository

### CODE REVIEW

File: api/internal/repository/get_friends_list_test.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+			ExpectedErr:    nil,
+			WantDBFail:     false,
+		},
+		{
+			Name:           "Internal Server Error",
+			UserEmail:      "already_exist_email_1@example.com",
+			ExpectedResult: nil,
+			ExpectedErr:    InternalErrorGetFriendList,
+			WantDBFail:     true,
+		},
```
Comment (nhan1603) [inline api/internal/repository/get_friends_list_test.go:41]: Can add another case of empty list of fiend email

Comment (TriAnh1995) [reply api/internal/repository/get_friends_list_test.go:41]: Personally I think "empty list of friends" is not and error case.
For example if a person that doesn't have any friends want to use this function, then the result should be an empty list, not an error message.

Comment (nhan1603) [reply api/internal/repository/get_friends_list_test.go:41]: Yes, it's not an error case. I agree that if a person that doesn't have any friends want to use this function, then the result should be an empty list. I think that you should add the case into unit test as "Success with empty friend list" for better understanding of the function if someone need to use it later.
